test/native-tests/test-char/sample.cpp

```diff
@@ -3,23 +3,7 @@
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 //-------------------------------------------------------------------------------------------------------
 
-#include "ChakraCore.h"
```

Comment:
this is the one causing the CI fails.. (no nullptr is defined.) will update after review.